Jenkinsfile_nightly

```diff
@@ -2,7 +2,13 @@
 
 properties([
 // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
-pipelineTriggers([cron('H 05 * * *')])
+pipelineTriggers([cron('H 05 * * *')]),
+parameters([
+  string(name: 'FRONTEND_URL', defaultValue: 'https://fis-ds-web.aat.platform.hmcts.net/', description: 'The URL you want to run tests against'),
```

Comment:
should we mention the params like edgecaseType?

Reply:
I would like to avoid any url issue for cross browser with trailing bit at the end and this is running smoke test to prove if sauce lab works and user can login to screens with different browsers and platforms, once that is proven , we can extend this to fully end to end application submission where edgecaseType will be required.